JSDemos/Demos/DataGrid/StatePersistence/description.md

```diff
@@ -1 +1,5 @@
-Sort and filter data, group, reorder and resize columns, change page numbers and page size. Once you are done, refresh the web page to&nbsp;see that the grid&rsquo;s state is&nbsp;automatically persisted to&nbsp;continue working from where you stopped.
\ No newline at end of file
+**DataGrid** state can be persistent - if a user sorts and filters data, groups, reorders and resizes columns or makes other changes, then the widget saves these modifications and restores them when a user reloads the page. The full list of modifications that can be saved is in the [stateStoring](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/) documentation.
+
+To enable state persistence, set **stateStoring**.[enabled](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/#enabled) to **true** and specify the [storageKey](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/#storageKey) and [type](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/#type) options. Depending on the type, the state can be saved to a <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage" target="_blank">localStorage</a> or <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage" target="_blank">sessionStorage</a>. With **localStorage**, the state persists across browser sessions; with **sessionStorage**, it is reset after the current session. This demo uses **localStorage**.
+
+Alternatively, you can create a custom storage and implement the [customSave](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/#customSave) and [customLoad](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/#customLoad) functions if you need to store data in accordance with a custom logic.
```

Comment:
"you can create a custom storage" - I have two remarks regarding this piece:
1) I would rather suggest that the developer already has a custom storage. Then, this phrase should read "if you want to save the state in a custom storage", but (see 2)
2) customSave and customLoad can be used without a custom storage, for example, to tweak the state a bit before saving it to localStorage. So, I think the following version would be more accurate:

```suggestion
Alternatively, you can implement the [customSave](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/#customSave) and [customLoad](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/stateStoring/#customLoad) functions to save and load the state according to a custom logic.
```

Reply:
@RomanTsukanov 
"create a custom storage" is erroneous here.
Since "customSave and customLoad can be used without a custom storage", should there be a sentence that it is possible to **use** a custom storage (this implies a change of the stateStoring's type) and **implement** these two functions?